src/renderers/shared/fiber/ReactFiber.js

```diff
@@ -122,13 +122,14 @@ function shouldConstruct(Component) {
 }
 
 // This is used to create an alternate fiber to do work on.
-exports.cloneFiber = function(fiber : Fiber) : Fiber {
+exports.cloneFiber = function(fiber : Fiber, priorityLevel : PriorityLevel) : Fiber {
   // We use a double buffering pooling technique because we know that we'll only
   // ever need at most two versions of a tree. We pool the "other" unused node
   // that we're free to reuse. This is lazily created to avoid allocating extra
   // objects for things that are never updated. It also allow us to reclaim the
   // extra memory if needed.
   if (fiber.alternate) {
+    fiber.alternate.pendingWorkPriority = priorityLevel;
```

Comment:
I guess this is okay since we bump the priority of all ancestors when enqueueing an update?